3rdparty/install.sh

```diff
@@ -0,0 +1,41 @@
+# install hts_engine_API
+pushd hts_engine_API/src
+touch ChangeLog
+export CFLAGS="$CFLAGS -fPIC"
+autoreconf -fvi
+./configure
+make clean
+make
+sudo make install
+popd
```

Comment:
Could you follow https://github.com/r9y9/nnsvs/blob/ce7a8f9186329eab4fb8567606ea49befb176439/.github/workflows/ci.yml#L28?